source/arch/metadatastore-format.rst

```diff
@@ -82,7 +82,7 @@ event. For example::
 
     {
         "uid" : <unique_id>,
-        "scan_id" : <non-unique-id>,
+        "scan_id" : <non-unique-id>, # is this specified to be an integer, or is anything valid here?
```

Comment:
+1


On Jan 29, 2015, at 11:10 AM, Thomas A Caswell <notifications@github.com<mailto:notifications@github.com>> wrote:


In source/arch/metadatastore-format.rst<https://github.com/NSLS-II/docs/pull/2#discussion_r23778463>:

> @@ -82,7 +82,7 @@ event. For example::
>
>      {
>          "uid" : <unique_id>,
> -        "scan_id" : <non-unique-id>,
> +        "scan_id" : <non-unique-id>, # is this specified to be an integer, or is anything valid here?


I would go with anything sortable

—
Reply to this email directly or view it on GitHub<https://github.com/NSLS-II/docs/pull/2/files#r23778463>.